src/components/TypeformSurvey/index.js

```diff
@@ -24,7 +24,9 @@ export const TypeformSurvey = InjectAppServices(
       fetchData();
     }, [dopplerLegacyClient]);
 
-    if (loading || surveyFormCompleted) {
+    const { isFreeAccount: isTrial } = appSessionRef.current.userData.user.plan;
+
+    if (loading || surveyFormCompleted || !isTrial) {
```

Comment:
No estoy seguro de si trial es el nombre correcto. Creo que podríamos evitarlo, no?

```suggestion
    const { isFreeAccount } = appSessionRef.current.userData.user.plan;

    if (loading || surveyFormCompleted) {
```

Reply:
Por lo que tengo entiendo, las cuentas Free son las que se utilizaban anteriormente (cuando no tenian vencimiento), ahora que se le puso tiempo (3 meses para su expiracion) paso a ser Trial.

Reply:
No tengo opinión fuerte. 

Tenés razón en que es el nombre correcto.

Si te parece que es más claro cambiarle el nombre en ese pequeño bloque, me parece bien.

Reply:
Otra opción, si queremos cambiar el nombre, es hacerlo en el mapper, y poner el nuevo nombre en todos lados.